app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource
+
+
+    def index
+      @ordered_events = Event.order(:title).all
```

Comment:
This fetche all available events, but do we want to show **all** Events? Or just the ones that belong to a conference, for which conference the user has the role organizer or cfp?